GoogleChromeLabs/ps-analysis-tool
# Remove Wappalyzer used for technologies from CLI

## Description
This PR aims to remove the wappalyzer package from the CLI thus deprecating the CLI's capability to analyse the technologies.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- In the terminal run `npm i` and then run `npm run build:cli:dashboard`.
- Now try to run technology analysis from CLI it should fail.
- And any analysis will not have technology analysis details.

## Additional Information:



## Screenshot/Screencast



---

## Checklist
